.github/workflows/basic.yml

```diff
@@ -32,6 +32,10 @@ jobs:
       - name: Musl build with default features
         run: |
           make LIBC=musl
+      
+      - name: s390x build with default features
+        run:
+          make ARCH=s390x
```

Comment:
don't you want to build also with `offline_fs_kbc` which is actually used for CC in s390x Secure Execution?